src/__tests__/test.js

```diff
@@ -1,7 +1,29 @@
-import { myModule } from '..';
+import fs from 'fs';
+import { join } from 'path';
 
-describe('test myModule', () => {
-  it('should return 42', () => {
-    expect(myModule()).toStrictEqual(42);
+import * as imageJs from 'image-js';
+import { IOBuffer } from 'iobuffer';
+
+import { readFile } from '../reader';
+import { createImage } from '../image';
+
+const testFiles = './data';
+describe('parse afm', () => {
+  it('write to file', () => {
+    const arrayBuffer = new IOBuffer(
+      fs.readFileSync(join(__dirname, `${testFiles}/raw data.001`)),
+    );
+    const read = readFile(arrayBuffer);
+    const image = createImage(read);
+    /* //uncomment to save files locally
```

Comment:
You could also add a test that compares part of the object (meta information like width, height but not the data itself).
It could be a to match snapshot but part of the test could also be done directly in the code using https://jestjs.io/docs/expect#tomatchobjectobject

Reply:
Metadata is contained in the header attribute of read in this instance.